admin/config.yml

```diff
@@ -54,6 +54,7 @@ collections:
       - { label: "Publish Date", name: "date", widget: "datetime" }
       - { label: "Tag", name: "areas", widget: "string" }
       - { label: "Featured Image", name: "thumbnail", widget: "image" }
-      - { label: "Thumbnail Description", name: "imgtext", widget: "string" }
+      - { label: "Featured Image Text", name: "imgtext", widget: "string" }
```

Comment:
yay for better naming in the cms fields! this will make the cms user experience way better!